<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Final</class>
 <widget class="QWidget" name="Final">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>实时界面</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>311</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>物品信息</string>
    </attribute>
    <widget class="QPushButton" name="car1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="source/apprcc.qrc">
       <normaloff>:/pic/car.jpg</normaloff>:/pic/car.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="car3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="source/apprcc.qrc">
       <normaloff>:/pic/car.jpg</normaloff>:/pic/car.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="car4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="source/apprcc.qrc">
       <normaloff>:/pic/car.jpg</normaloff>:/pic/car.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="car8">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="source/apprcc.qrc">
       <normaloff>:/pic/car.jpg</normaloff>:/pic/car.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="car7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="source/apprcc.qrc">
       <normaloff>:/pic/car.jpg</normaloff>:/pic/car.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="car9">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="source/apprcc.qrc">
       <normaloff>:/pic/car.jpg</normaloff>:/pic/car.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="car6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="source/apprcc.qrc">
       <normaloff>:/pic/car.jpg</normaloff>:/pic/car.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="car2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="source/apprcc.qrc">
       <normaloff>:/pic/car.jpg</normaloff>:/pic/car.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="car5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="source/apprcc.qrc">
       <normaloff>:/pic/car.jpg</normaloff>:/pic/car.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>324</width>
     <height>316</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>实时信息</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>物品类别：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QComboBox" name="Class"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>位置：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QComboBox" name="location"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>湿度：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="hum"/>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>温度：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLineEdit" name="temp"/>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>光照：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QComboBox" name="light"/>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>碰撞：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QComboBox" name="Click"/>
     </item>
     <item row="6" column="4">
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="update">
       <property name="text">
        <string>更改</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="source/apprcc.qrc">:/pic/light.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="source/apprcc.qrc">:/pic/temp.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="source/apprcc.qrc">:/pic/hum.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="source/apprcc.qrc">:/pic/click.jpg</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="MyMessage">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>我的信息</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd HH:mm:ss</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="source/apprcc.qrc"/>
 </resources>
 <connections/>
</ui>
